pagopa/io-react-native-login-utils
# [IOCOM-1429] Configurable ephemeral session on iOS

## Short description
This PR makes the `prefersEphemeralWebBrowserSession` property configurable from typescript on iOS.
It also updates react-native to 0.71.19 in order to compile on newer xCode

## List of changes proposed in this pull request
- New property `shareiOSCookies` to set `prefersEphemeralWebBrowserSession` value, only on iOS
- Such property has a default value set to `false` for retrocompatibility
- Android is unaffected, since there is no way to disable such behaviour and cookies are always shared
kotlin's methods updated with no practical effect in order not to throw at runtime
- react-native updated to 0.71.19

## How to test
Using the example app, open an Url to a website where you have an authenticated session on Safari:
- with a value of `true`, you should find yourself logged in
- with a value of `false` (or a default value), you should find yourself not authenticated

Be aware that this works only on a pysical device. It does not work on the simulator (i.e., cookies are never shared on the simulator)
